k-distribution/tests/regression-new/context-alias/test.k

```diff
@@ -10,12 +10,12 @@ context alias [heatable]: <k> HERE ...</k> <stuff> true </stuff>
 
 context alias [heatable2]: <k> HERE ...</k> <shouldHeat> B </shouldHeat> requires B
 
-syntax Expr ::= Expr "+" Expr [strict(heatable)]
```

Comment:
I'd like us to wait to merge this PR until my PR fixing this particular set of tests in a better way is merged.